packages/feedback/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      imports: [
+        RouterTestingModule
+      ],
+      declarations: [
+        AppComponent
+      ],
+    }).compileComponents();
+  }));
```

Comment:
And remove the closing parenthesis after the above change. (At line 6)

```suggestion
  });
```